docs/pages/web/buttontoggle.tsx

```diff
@@ -0,0 +1,321 @@
+      </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description={`To make it clearer, you may want to change the label text to indicate that the ButtonToggle has been selected. For instance, changing "follow" to "Following."`}
+            sandpackExample={
+              <SandpackExample
+                code={changeLabelsDo}
+                hideEditor
+                name={`To make it clearer, you may want to change the label text to indicate that the ButtonToggle has been selected. For instance, changing "follow" to "Following."`}
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description="Make sure that the ButtonToggle(s) in your application are consistently styled and placed. This should also apply to their sizing, maintaining uniformity throughout the experience."
+            sandpackExample={
+              <SandpackExample
+                code={consistencyDo}
+                hideEditor
+                name="Make sure that the ButtonToggle(s) in your application are consistently styled and placed. This should also apply to their sizing, maintaining uniformity throughout the experience."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description="Place the ButtonToggle(s) in a location where users would naturally expect to find them, taking into consideration the context. For instance, position it next to a related feature."
+            sandpackExample={
+              <SandpackExample
+                code={locationDo}
+                hideEditor
+                name="Place the ButtonToggle(s) in a location where users would naturally expect to find them, taking into consideration the context. For instance, position it next to a related feature."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+            type="do"
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description="If the ButtonToggle(s) trigger a significant action or irreversible change, it is recommended to include a confirmation, such as a toast message."
+            sandpackExample={
+              <SandpackExample
+                code={confirmationDo}
+                hideEditor
+                name="If the ButtonToggle(s) trigger a significant action or irreversible change, it is recommended to include a confirmation, such as a toast message."
```

Comment:
can we replace names with a shorter name? In all examples.

Reply:
DONE